backend/src/routers/borrowRouter.ts

```diff
@@ -0,0 +1,81 @@
+import { Router } from 'express';
+import { z } from 'zod';
+import { validateRequestBody } from 'zod-express-middleware';
+import { borrowGame, returnGame } from '../services/borrowService.js';
+
+const borrowRouter = Router();
+
+const borrowSchema = z.object({
+	gameId: z.string().min(1),
+	user: z.string().min(1),
+	borrowStart: z.string().datetime(),
+	borrowEnd: z.string().datetime()
+});
+
+/**
+ * @api {post} /api/v1/games/borrow Borrow a game
```

Comment:
Correct endpoint in apidoc
```suggestion
 * @api {post} /api/v1/borrow Borrow a game
```